cmstestsuite/Tests.py

```diff
@@ -251,54 +254,52 @@
 
     Test('managed-correct',
          task=batch_fileio_managed, filenames=['managed-correct.%l'],
-         languages=(LANG_C, LANG_CPP, LANG_PASCAL, LANG_PYTHON3, LANG_JAVA,
-                    LANG_C_SHARP),
+         languages=COMMUNICATION_LANGUAGES + (LANG_C_SHARP,),
          checks=[CheckOverallScore(100, 100)],
          user_tests=True, user_managers=['grader.%l'],
          user_checks=[CheckUserTestEvaluated()]),
 
     Test('managed-incorrect',
          task=batch_fileio_managed, filenames=['managed-incorrect.%l'],
-         languages=(LANG_C, LANG_CPP, LANG_PASCAL, LANG_PYTHON3, LANG_JAVA,
-                    LANG_C_SHARP),
+         languages=COMMUNICATION_LANGUAGES + (LANG_C_SHARP,),
          checks=[CheckOverallScore(0, 100)]),
 
     # Communication tasks. PHP is not yet supported.
 
     Test('communication-fifoio-correct',
          task=communication_fifoio_stubbed,
          filenames=['communication-stubbed-correct.%l'],
-         languages=(LANG_C, LANG_CPP, LANG_PASCAL, LANG_PYTHON3, LANG_JAVA),
```

Comment:
Not having C# here is on purpose or maybe was it forgotten at the time?

Reply:
If it's not too tedious it would probably be good to implement `communication` solutions for all languages.

Reply:
agreed, left a todo for it.